pkg/sif/sif.go

```diff
@@ -135,6 +135,7 @@ const (
 	DataCryptoMessage                          // cryptographic message data object
 	DataSBOM                                   // software bill of materials
 	DataOCIBlob                                // oci blob data object
+	DataOCIImageIndex                          // oci image index data object
```

Comment:
I'm not sure what this _should_ be called. On the one hand, `DataOCIImageIndex` is certainly accurate, as it will contain a `application/vnd.oci.image.index.v1+json`. On the other hand, there may also be `DataOCIBlob`s of type `application/vnd.oci.image.index.v1+json`, which is somewhat confusing.

 I wonder if it'd be worth choosing a name for this that makes its purpose of being a root index (versus a child index) clear? What do you think about:

```suggestion
	DataOCIRootIndex                           // root OCI index
```